not-to-release/sources/reviews/018548.xml.conllu

```diff
@@ -196,7 +196,7 @@
 7	and	and	CCONJ	CC	_	9	cc	9:cc	_
 8	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	9	nsubj	9:nsubj	_
 9	deals	deal	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	2	conj	2:conj	_
-10	crushing	crush	VERB	VBG	VerbForm=Ger	11	amod	11:amod	_
```

Comment:
_blows that crush something_ sounds a bit odd but I think it's nevertheless grammatical, so I'd say `VBG` is correct here.